src/lib.rs

```diff
@@ -66,15 +68,32 @@ use {
   prometheus_metrics::serve_prometheus,
 };
 
+#[derive(Parser, Debug)]
+struct CmdArgs {
+  #[arg(long, default_value_t = false)]
+  /// if you start multiple lemmy server instances set this to true on all but one of them
+  disable_scheduled_tasks: bool,
+  /// set to false to disable the http server
```

Comment:
```suggestion
  /// Whether or not to run the HTTP server.
  ///
  /// This can be used to run a Lemmy server process that only runs scheduled tasks.
```